staging/prometheus-operator/Chart.yaml

```diff
@@ -1,20 +1,49 @@
-apiVersion: v1
-description: Provides easy monitoring definitions for Kubernetes services, and deployment and management of Prometheus instances.
+apiVersion: v2
+description: prometheus-operator collects Kubernetes manifests, Grafana dashboards, and Prometheus rules combined with documentation and scripts to provide easy to operate end-to-end Kubernetes cluster monitoring with Prometheus using the Prometheus Operator.
 icon: https://raw.githubusercontent.com/prometheus/prometheus.github.io/master/assets/prometheus_logo-cb55bb5c346.png
 engine: gotpl
+type: application
 maintainers:
   - name: vsliouniaev
   - name: bismarck
   - name: gianrubio
     email: gianrubio@gmail.com
+  - name: gkarthiks
+    email: github.gkarthiks@gmail.com
+  - name: scottrigby
+    email: scott@r6by.com
+  - name: Xtigyro
+    email: miroslav.hadzhiev@gmail.com
 name: prometheus-operator
```

Comment:
just checking - does leaving the chart name `prometheus-operator` allow us to sidestep issues that come along with renaming the chart? are those primarily upgrade issues?

Reply:
yep it does :)